cic/clsp/drop_coins/rekey_completion.clsp

```diff
@@ -27,7 +27,7 @@
         (list ASSERT_MY_PARENT_ID (calculate_coin_id parent_parent parent_puzzle parent_amount))
         (list ASSERT_PUZZLE_ANNOUNCEMENT (sha256 parent_puzzle REKEY_KW))
         (list CREATE_PUZZLE_ANNOUNCEMENT REKEY_KW)
-        (list CREATE_COIN_ANNOUNCEMENT '$') ; for fees
+        (list CREATE_COIN_ANNOUNCEMENT ()) ; for fees
```

Comment:
this doesn't save any space `()` compiles to `80` and `$` to `24`, what's the reason?

Reply:
$ actually compiles to `8124` so it does save us a byte